src/EtwClrProfiler/CorProfilerTracer.cpp

```diff
@@ -119,7 +119,7 @@ EXTERN_C void __stdcall TailcallMethod(FunctionID functionID)
 #if defined(_M_IX86)
 // see http://msdn.microsoft.com/en-us/library/4ks26t93.aspx  for inline assembly.   Not supported on X64.   
 
-void __declspec( naked ) __stdcall EnterMethodNaked(FunctionID funcID)
```

Comment:
@vancem I can no longer reproduce the specific case where this failure occurred. It's back to using the code as originally written in this file. Very strange.